### ISSUE TITLE
Display app versions

### ISSUE THREAD

**Issue by rodionovd** [argon/mas#43]
## `list` command

Current output:

```
409203825 Numbers
847496013 Deckset
1024069033 Veertu
452840086 Receigen
409201541 Pages
```

Proposed output:

```
409203825 Numbers (3.6)
847496013 Deckset (1.6.2)
1024069033 Veertu (1.1.4)
452840086 Receigen (3.4.0)
409201541 Pages (5.6)
```

## `outdated` command

Current output:

```
409203825 Numbers (3.6.2)
409201541 Pages (5.6.2)
```

Proposed output:

```
409203825 Numbers (3.6 -> 3.6.2)
409201541 Pages (5.6 -> 5.6.2)
```

----

I'm working on a PR to implement this feature.
*Question:* Should I hide this feature under a new flag (e.g. `--version`) or just make it a default behaviour for both commands?

**Comment by argon** [comment]
Just make it the default behaviour, I think

—A

**Comment by rodionovd** [comment]
OK, will do.

Oh, I'll probably postpone the PR until #44 is merged (I haven't even upgraded to Xcode 8 yet)

**Comment by argon** [comment]
I'm happy to wait for your PR to merge #44. I'm still improving some things. it's a work in progress.

**Comment by rodionovd** [comment]
Implemented in #45
